packages/web3torrent/.env

```diff
@@ -1,15 +1,2 @@
-# TODO: Create React App doesn't like that the wallet is using other versions of the same packages
-# By setting this to true we skip that check (for now )
-SKIP_PREFLIGHT_CHECK=true
-
-REACT_APP_WALLET_URL=http://localhost:3055
-REACT_APP_FAKE_CHANNEL_PROVIDER=false
-
-REACT_APP_CHAIN_NETWORK_ID=9001
-CHAIN_NETWORK_ID=9001
-
-REACT_APP_FIREBASE_PREFIX='web3t'
-REACT_APP_FIREBASE_API_KEY='AIzaSyAOvhDzJir_El3O6SJ2xQlrpOisnObq6zw'
-
-REACT_APP_TRACKER_URL=web3torrent-tracker-staging.herokuapp.com
-REACT_APP_TRACKER_URL_HTTP_PROTOCOL=https
+# Ignore this file. react-scripts will read it automatically, but we don't want that.
```

Comment:
Can we improve this sentence? Do we need the file to prevent react-scripts throwing an error? We don't want react-scripts to read it automatically... but it still will? How about: 
```suggestion
# This file should remain empty -- it exists because react-scripts requires it to. Do not set environment variables here. 
```